dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.42/V2_42_28__Make_ou_event_and_enrollment_not_null.sql

```diff
@@ -0,0 +1,61 @@
+-- Delete invalid events linked to invalid enrollments.
+delete from event e where e.enrollmentid in (
```

Comment:
What about foreign key constraints of the eventid existing in other tables?

Reply:
:fearful: 
I completely forgot any of this and I got lucky in the SL database when applying this.
I am going to add all the necessary checks on other tables, I will probably split this migration in 2 then.

Reply:
Will this be executed for all instances, regardless of whether they have the inconsistency? Would it be more sensible to first check for any inconsistencies before proceeding with the procedure?